crypto/fipsmodule/ec/ec_test.cc

```diff
@@ -211,6 +211,19 @@ static const uint8_t kP224PublicKey_compressed_0x02[] = {
   0xc5, 0x82, 0x78, 0x85
 };
 
+static const uint8_t kP224PublicKey_hybrid_0x02[] = {
```

Comment:
where did you get these values?

Reply:
I ran the test vectors in the same file against [the brief test program](https://github.com/aws/aws-lc/pull/1936#issue-2598629333) in the PR description, when compiled against OpenSSL.